### DISCUSSION TITLE
Add-to-calendar button into the template email!

### DISCUSSION THREAD

**Discussion by mrc-bsllt** [mjmlio/mjml#2628] [Q&A]
Hello everyone!
Can anyone tell me if it's possible, somehow, to insert a button to add an event to the calendar? 
Thanks!

**Comment by rustemia98** [comment]
Hi, maybe this can help you.
`<mjml>
  <mj-head>
    <mj-style>
      .button {
        background-color: #3498db;
        border: none;
        color: white;
        padding: 15px 32px;
        text-align: center;
        text-decoration: none;
        display: inline-block;
        font-size: 16px;
        margin: 4px 2px;
        cursor: pointer;
      }
      .button-google {
        background-color: #DB4437;
      }
      .button-outlook {
        background-color: #0072C6;
      }
      .button-ical {
        background-color: #4CAE4C;
      }
    </mj-style>
  </mj-head>
  <mj-body>
    <mj-section>
      <mj-column>
        <mj-button href="https://www.google.com/calendar/render?action=TEMPLATE&text=Example+Event&dates=20220101T010000Z/20220101T020000Z&details=Example+description&location=Example+Location&sf=true&output=xml#eventpage_6" css-class="button button-google">Add to Google Calendar</mj-button>
        <mj-button href="data:text/calendar;charset=utf8,BEGIN:VCALENDAR%0AVERSION:2.0%0APRODID:-//My Company//NONSGML Event//EN%0AURL:http://www.example.com%0AUID:1234567890abcdef%0ADTSTART:20220101T010000Z%0ADTEND:20220101T020000Z%0ASUMMARY:Example Event%0ADESCRIPTION:Example description%0ALOCATION:Example Location%0AEND:VEVENT%0AEND:VCALENDAR" css-class="button button-outlook">Add to Outlook</mj-button>
        <mj-button href="data:text/calendar;charset=utf8,BEGIN:VCALENDAR%0AVERSION:2.0%0APRODID:-//My Company//NONSGML Event//EN%0AURL:http://www.example.com%0AUID:1234567890abcdef%0ADTSTART:20220101T010000Z%0ADTEND:20220101T020000Z%0ASUMMARY:Example Event%0ADESCRIPTION:Example description%0ALOCATION:Example Location%0AEND:VEVENT%0AEND:VCALENDAR" css-class="button button-ical">Add to Apple Calendar</mj-button>
      </mj-column>
    </mj-section>
  </mj-body>
</mjml>
`
